python/lsst/jointcal/jointcal.py

```diff
@@ -25,8 +24,11 @@
 
 __all__ = ["JointcalConfig", "JointcalTask"]
 
+Photometry = collections.namedtuple('Photometry', ('fit', 'model'))
+Astrometry = collections.namedtuple('Astrometry', ('fit', 'model', 'sky_to_tan_projection'))
 
-class JointcalRunner(pipeBase.TaskRunner):
+
+class JointcalRunner(pipeBase.ButlerInitializedTaskRunner):
```

Comment:
Also, pipeBase says TaskRunner must be picklable for multi-processing and if not then do you want to set canMultiprocess = False in JointcalTask?  That is what is done in MosaicTask, according to the docstring, which also adds confusion.